pkg/util/timeutil/pgdate/field_extract.go

```diff
@@ -263,6 +270,14 @@ func (fe *fieldExtract) interpretNumber(numbers []numberChunk, idx int, textMont
 			// BUT NOT: 1999 1
 			return fe.SetDayOfYear(chunk)
 
+		case fe.Wants(fieldMinute) && chunk.v <= 60 && chunk.v >= 0:
+			// db2 timestamp allows for minutes to recorded with periods.
+			// Example 13.12.50 (hh.mm.ss)
+			fe.isDB2 = true
```

Comment:
should `fe.isDB2` already be set by the `-` separator below, or is the `-` between the YYYY-MM-DD and HH.MM.SS optional?

Reply:
You could have something like `yyyy-mm-dd hh.mm.ss` which is a DB2 type, we need to set DB2 on `mm` so if we get something like `yyyy-mm-dd hh.mm:ss` we could throw an error. 

In this cause it won't `-` but still should be treated as db2 timestamp.